lib/github-pages-health-check/cloudflare.rb

```diff
@@ -1,9 +1,10 @@
-class GitHubPages
-  class HealthCheck
+module GitHubPages
```

Comment:
This might collide with other `GitHubPages` modules or classes... I think it's a class in some places (like shell)?